build.gradle

```diff
@@ -11,7 +11,7 @@ buildscript {
     }
 
     dependencies {
-        classpath "com.android.tools.build:gradle:$gradlePluginVersion"
+        classpath 'com.android.tools.build:gradle:8.5.1'
```

Comment:
Why didn't you just bump the variable?

Reply:
Good Q. It was because I wanted AS to automate the migration for me (it doesn't support automatic migration if you have a variable reference). Let me change back the variable reference.